drivers/include/lsm6ds3.h

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
If we stay with the (below discussed) decision that you are the only author, please replace FU by HAW